src/pages/my-patches/StatusSelector.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import {
+  MyPatchesQueryParams,
+  PatchStatus,
+  ALL_PATCH_STATUS,
+} from "types/patch";
+import { TreeSelect } from "components/TreeSelect";
+import { useStatusesFilter } from "hooks";
+
+export const StatusSelector = () => {
+  const [statusVal, statusValOnChange] = useStatusesFilter(
+    MyPatchesQueryParams.Statuses
+  );
+
+  return (
+    <TreeSelect
+      onChange={statusValOnChange}
+      state={statusVal}
+      tData={treeData}
+      inputLabel="Patch status:  "
+      dataCy="test-status-select"
+      width="25%"
+    />
+  );
+};
+
+const treeData = [
+  {
+    title: "All",
+    value: ALL_PATCH_STATUS,
+    key: ALL_PATCH_STATUS,
+  },
+  {
+    title: "Created",
+    value: PatchStatus.Created,
+    key: PatchStatus.Created,
+  },
+  {
+    title: "Started",
+    value: PatchStatus.Started,
+    key: PatchStatus.Started,
+  },
+  {
+    title: "Success",
```

Comment:
I realize it's wrong in the db models, but could we make the tense in the tiles consistent? So that we have either success/fail or succeeded/failed. The task status select has the same thing so feel free to defer this and I can make a ticket